packages/procaptcha/package.json

```diff
@@ -7,9 +7,9 @@
     "type": "module",
     "types": "./dist/index.d.ts",
     "scripts": {
-        "test": "NODE_ENV=test ts-mocha \"tests/**/*.test.ts\" && exit 0",
         "clean": "tsc --build --clean",
-        "build": "tsc --build --verbose tsconfig.json",
+        "prebuild": "node prebuild-write-version.js",
```

Comment:
I __think__ this conflicts with https://docs.npmjs.com/cli/v10/using-npm/scripts#npm-run-user-defined

You can probably make use of the above instead, i.e. don't worry about calling the prebuild script yourself because npm should do it for you

Reply:
yeah this works